### PR TITLE
feature: pull sandbox image if it doesn't exist and complete basic sandbox operation

### PR DESCRIPTION
**Pull request by YaoZengzeng**
Signed-off-by: YaoZengzeng <yaozengzeng@zju.edu.cn>

**1.Describe what this PR did**

1. Pull the sandbox image if it doens't exist.

2. Integrate "basic sandbox operation" test into CI.

### CODE REVIEW

**Comment by YaoZengzeng** [comment]
@allencloud @Letty5411 @czm4514 PTAL.

**Comment by Letty5411** [comment]
LGTM

File: daemon/mgr/cri_utils.go [hunk 1]
```diff
@@ -365,3 +367,24 @@ func imageToCriImage(image *apitypes.ImageInfo) (*runtime.Image, error) {
 		Size_:       size,
 	}, nil
 }
+
+// ensureSandboxImageExists pulls the image when it's not present.
+func (c *CriManager) ensureSandboxImageExists(ctx context.Context, image string) error {
+	_, err := c.ImageMgr.GetImage(ctx, image)
+	// TODO: maybe we should distinguish NotFound error with others.
```
Comment (allencloud) [inline daemon/mgr/cri_utils.go]: Yes, only NotFound error shall we pull the image, right? @YaoZengzeng

Comment (YaoZengzeng) [reply daemon/mgr/cri_utils.go]: @allencloud  yes 😄

**Pull request merged**